src/main/java/frc/team2412/robot/subsystem/ClimbSubsystem.java

```diff
@@ -208,4 +239,27 @@ public void simulationPeriodic() {
         climbDynamicMotor.getSimCollection().setIntegratedSensorRawPosition((int) (motorDynamicSpeed / timeElapsed));
         lastUpdatedTime = timeNow;
     }
+
+    @Log
+    public double encoderPOsition() {
+        return climbFixedMotor.getSelectedSensorPosition();
+    }
+
+    @Config
+    public void resetEncoder(boolean hfh) {
+        if (hfh) {
+            climbFixedMotor.setSelectedSensorPosition(0);
+        }
+    }
+
+    @Config(name = "PID")
+    private void setPID(@Config(name = "P", defaultValueNumeric = P) double p,
+            @Config(name = "I", defaultValueNumeric = I) double i,
+            @Config(name = "D", defaultValueNumeric = D) double d) {
+        climbFixedMotor.config_kP(0, p);
+        climbFixedMotor.config_kI(0, i);
+        climbFixedMotor.config_kD(0, d);
+    }
+
+    // spotless:on
```

Comment:
Spotless should already be on, this can be deleted (looks like a remnant from disabling spotless earlier).